domain/src/main/java/com/jakefoundation/buildabotworkshop/domain/Owner.java

```diff
@@ -0,0 +1,23 @@
+
+/* Create a class named Owner in Domain and implement it according to this UML.
+ * Let private fields be final.
+ * */
+
+public class Owner {
```

Comment:
looks like this also resolves #24. Please add that to your PR description :)

Reply:
how do I do that again?